jlm/llvm/opt/alias-analyses/PointerObjectSet.cpp

```diff
@@ -1398,6 +1403,7 @@ PointerObjectConstraintSet::RunWorklistSolver(WorklistStatistics & statistics)
   if constexpr (EnableOnlineCycleDetection)
   {
     static_assert(!EnableHybridCycleDetection, "OnlineCD can not be combined with HybridCD");
+    static_assert(!EnableLazyCycleDetection, "OnlineCD can not be combined with LazyCD");
   }
```

Comment:
What about the combination of Lazy cycle detection and hybrid cycle detection? This is not checked for, is it?

Reply:
Or can both be enabled at the same time?

Reply:
It seem to look like it, looking at the rest of the code...

Reply:
Yes, they can be combined. They are also from the same paper (Hardekopf 2007)